lib/llvm-backend/cpp/object_loader.hh

```diff
@@ -49,9 +49,7 @@ typedef struct {
   visit_fde_t visit_fde;
 } callbacks_t;
 
-typedef struct {
-  size_t data, vtable;
-} box_any_t;
+using runtime_error_t = void*;
```

Comment:
```suggestion
typedef struct runtime_error_t_privdecl *runtime_error_t;
```
will get you a typesafe pointer-to-opaque, where the opaque is a forward declaration of a struct that is never defined.

However, maybe this type shouldn't be a pointer, and all the references to `runtime_error_t` should really be `runtime_error_t*`? You could write `struct runtime_error_t;` here and never provide it a body. Is that closer to what's really going on?

Reply:
Hmm, I have mixed feelings about it, other than the explicit `copy` function exported from Rust, the C++ code doesn't need to know anything about this type... that copy function does change things though